POC/ColorPicker/ColorPicker/MainWindow.xaml.cs

```diff
@@ -20,9 +21,29 @@ namespace ColorPicker
     /// </summary>
     public partial class MainWindow : Window
     {
+        private Random _random = new Random(); // For testing only
+
         public MainWindow()
         {
             InitializeComponent();
+            this.DataContext = this;
```

Comment:
Not sure what this does

Reply:
This was removed - it was indeed unnecessary.